ding/model/template/efficientzero/efficientzero_tictactoe_model.py

```diff
@@ -0,0 +1,607 @@
+"""
```

Comment:
remove gomoku related codes like model and config now

Reply:
ok